core/src/ast/mod.rs

```diff
@@ -224,9 +224,10 @@ impl ToSql for Statement {
                 column: _column,
             } => {
                 format!(
-                    "CREATE INDEX {} ON {} (..order_by_expr..)",
+                    "CREATE INDEX {} ON {} {}",
                     name.to_sql(),
-                    table_name.to_sql()
+                    table_name.to_sql(),
+                    _column.to_sql()
```

Comment:
Now the `_column` is being used, so we can remove `_` underscore prefix.
`column.to_sql()` would be enough